tools/dev/setup-toolchain.sh

```diff
@@ -28,13 +28,15 @@ wget "http://ftp.gnu.org/gnu/binutils/binutils-2.25.tar.bz2"
 wget "http://ftp.gnu.org/gnu/gcc/gcc-5.3.0/gcc-5.3.0.tar.bz2"
 
 # Get required packages.
-apt-get install g++
+apt-get install -y g++ doxygen genisoimage gdb
 
 # Export variables.
 export PREFIX=/usr/local/cross
 export TARGET=i386-elf
+export CFLAGS=-pipe
+export CXXFLAGS=-pipe
 sh -c "echo 'export TARGET=$TARGET' > /etc/profile.d/var.sh"
-sh -c "echo 'export PATH=$PATH:$PREFIX/bin' >> /etc/profile.d/var.sh"
+sh -c "echo 'export PATH=\$PATH:$PREFIX/bin' >> /etc/profile.d/var.sh"
```

Comment:
I did not understand the question.

This change basically causes `export PATH=$PATH:$PREFIX/bin` to be saved in `/etc/profile.d/var.sh`, instead of resolving `$PATH` based on the environment where `setup-toolchain.sh` ran, fixing issues where the environment defined by `sudo` is different from those where the user really is (which is my case).

An basic example: If someone has [pyenv](https://github.com/pyenv/pyenv) installed, loaded by `.bashrc` **before** using something like `source /etc/profile.d/var.sh` (which is my case), this change allow `$PREFIX/bin` to be *appended* to `PATH`, instead of replacing `PATH` with the `PATH` that `sudo` uses followed by `$PREFIX/bin`.

It's clear now?